sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -163,7 +163,7 @@ case class ExplainCommand(
 
   // Run through the optimizer to generate the physical plan.
   override def run(sparkSession: SparkSession): Seq[Row] = try {
-    val outputString = sparkSession.sessionState.executePlan(logicalPlan).explainString(mode)
+    val outputString = sparkSession.sessionState.executePlan(logicalPlan, true).explainString(mode)
```

Comment:
The `logicalPlan` is not a command, right?

Reply:
Let's use
```
object CommandExecutionMode extends Enumeration {
  val COMMON, EAGERLY, SKIP = Value
}
```